docs/architecture/adr-001-abci++-adoption.md

```diff
@@ -1,5 +1,9 @@
 # ADR 001: ABCI++ Adoption
 
+## Status
+
+The proposed and initial implementation is complete.
```

Comment:
What does that mean? Is it accepted or only proposed?

Reply:
Resolved by 34ed8282e7571fe3c105799bdb7840785af39f4e